### PR TITLE
chore(DOCS): add multicast

### PR DESCRIPTION
**Pull request by davidchase**

### CODE REVIEW

**Comment by davidchase** [comment]
woo hooo ship them all 👍

File: docs/api.rst [hunk 1]
```diff
@@ -1064,6 +1064,25 @@ Recover from a stream failure by calling a function to create a new :ref:`Stream
...
+.. _multicast:
+
+multicast
+`````````
+
+.. code-block:: haskell
+
+  multicast :: Stream a -> Stream a
+
+Returns a :ref:`Stream` equivalent to the original, but which can be shared more efficiently among multiple consumers. ::
```
Comment (Frikki) [inline docs/api.rst]: Corrrect:
> Returns a :ref:`Stream` equivalent to the original but which can be shared more efficiently among multiple consumers.

We don’t have two independent clauses, so no comma before _but_. Even without _but_, we have an adjective clause that modifies _Stream_; _which_ is a relative pronoun referring to the antecedent _Stream_.

File: docs/api.rst [hunk 2]
```diff
@@ -1064,6 +1064,25 @@ Recover from a stream failure by calling a function to create a new :ref:`Stream
...
+
+  stream:             -a-b-c-d->
+  multicast(stream):  -a-b-c-d->
+
+Using multicast allows you to build up a stream of maps, filters, and other transformations, and then share it efficiently with multiple observers.
```
Comment (Frikki) [inline docs/api.rst]: Discard _Using_:
> Multicast allows you to build up a stream of maps, filters, and other transformations, and then share it efficiently with multiple observers.

**Comment by Frikki** [comment]
I’m concerned about the wording "... can be shared more efficiently among multiple consumers." Is it even feasible to share a non-multicast stream with multiple consumers?

**Comment by Frikki** [comment]
I am content now with the current wording. This example by David demonstrates: https://www.webpackbin.com/bins/-KunuEEJxMtd-L48fPBk

**Pull request merged**